lib/puppet/provider/login.rb

```diff
@@ -16,26 +16,59 @@
 
 require 'json'
 
+ONEVIEW_AUTH_FILE = 'ONEVIEW_AUTH_FILE'.freeze
+ONEVIEW_URL = 'ONEVIEW_URL'.freeze
+I3S_AUTH_FILE = 'I3S_AUTH_FILE'.freeze
```

Comment:
We should either use IMAGE_STREAMER or I3S on all instances, mixing them both is likely not a good scenario. I'd go with IMAGE_STREAMER_*